packages/core/src/part.js

```diff
@@ -357,20 +357,68 @@ Part.prototype.generateTransform = function(transforms) {
   }
 }
 
-/** Chainable way to set the grain property */
-Part.prototype.setGrain = function (grain = 90) {
-  this.attributes.set('data-grain', grain)
+/** Chainable way to add the cut info */
+Part.prototype.addCut = function (cut=2, material='fabric', identical=false) {
```

Comment:
change to `cut=1` I don't think it's safe to assume most pieces are pairs

Reply:
It's a default, and defaults should be sensible. 
My feeling is that cut=2 is by far the most common option.

Quick check for designs in the first three letters of the alphabet seems to confirm that feeling:

| Design | Cut=1 | Cut=2 |
| --------- | -------- | --------- |
| Aaron | 2 | 0 |
| Albert | 3 | 0 |
| Bee | 1 | 2 |
| Benjamin | 1 | 1 |
| Bent | 0 | 4 |
| Bob | 1 | 0 |
| Breanna | 0 | 3 |
| Brian | 0 | 3 |
| Bruce | 1 | 3 |
| Carton | 2 | 14 |
| Carlita | 2 | 15 |
| Cathrin | 0 | 6 |
| Charlie | 2 | 7 |
| Cornelius | 2 | 6 ||
| **Summary** | 17 | 64 |

So leaving 2 as I feel it's the most sensible default.